util/src/main/java/tech/pegasys/pantheon/util/enode/EnodeURL.java

```diff
@@ -246,18 +286,92 @@ public Builder ipAddress(final String ip) {
       return this;
     }
 
-    public Builder listeningPort(final Integer listeningPort) {
-      this.listeningPort = listeningPort;
+    public Builder ports(final int listeningAndDiscoveryPorts) {
```

Comment:
I think this method name doesn't clearly communicate the outcome of using it. Either we remove it (and force the user to use `listeningPort()` and `discoveryPort()` or we rename it to something a bit clearer.
e.g. `discoveryAndListeningPorts()`

Reply:
👍 updated